Library/UIKit/UIViewController+StoryboardSegueIdentifierProtocol.swift

```diff
@@ -9,16 +9,29 @@
 import Foundation
 import UIKit
 
-public extension UIViewController {
+public protocol UIViewControllerType {
```

Comment:
I don't really like the name of this protocol and the empty extension below. Could we change it to just an extension of UIViewController?